firebase-vscode/src/options.ts

```diff
@@ -26,7 +26,6 @@ const defaultOptions: Readonly<VsCodeOptions> = {
   projectNumber: "",
   projectRoot: "",
   account: "",
-  json: true,
```

Comment:
The VSCE right now doesn't call Firestore commands. So I believe this should be a noop.